Automattic/simplenote-macos
# Simperium Initialization Sequence

### Fix
In this PR we're updating the init sequence, so that Simperium (and the CoreData Stack) are initialized before the Interface itself.

Since we're moving over to NSFetchedResutlsController, this step is required, in order to avoid initialization issues.

cc @eshurakov (Thank you sir!!)

Ref. #626


### Test
0. **Update** your local `.mobile-secrets` checkout please!
1. Launch Simplenote (logged off state)

- [x] Verify the Simplenote Icon shows up correctly
- [x] Verify the Login Screen looks exactly the same as ever
- [x] Verify you can log into your account (and your notes do show up!)

### Review
These changes do not require release notes.
